### ISSUE TITLE
hotfix演示代码没有按预期读出

### ISSUE THREAD

**Issue by liunixgithub** [FunAudioLLM/CosyVoice#1905]
样例代码example.py中演示# hotfix usage的部分有问题，没有按预期读出
样例代码问题部分：
for i, j in enumerate(cosyvoice.inference_zero_shot('高管也通过电话、短信、微信等方式对报道[j][ǐ]予好评。', 'You are a helpful assistant.<|endofprompt|>希望你以后能够做的比我还好呦。',
                                                        '/code/pipeline/lib/CosyVoice/asset/zero_shot_prompt.wav', stream=False)):
        torchaudio.save('hotfix_{}.wav'.format(i), j['tts_speech'], cosyvoice.sample_rate)
语音读出信息为：

[hotfix_0.wav](https://github.com/user-attachments/files/28776429/hotfix_0.wav)

高管也通过电话、短信、微信等方式对报道j予好评

输出信息：
[2026-06-10 09:03:57,276] [WARNING] [real_accelerator.py:162:get_accelerator] Setting accelerator to CPU. If you have GPU or other accelerator, we were unable to detect it.
[2026-06-10 09:03:57,282] [INFO] [real_accelerator.py:203:get_accelerator] Setting ds_accelerator to cpu (auto detect)
Sliding Window Attention is enabled but not implemented for `sdpa`; unexpected results may be encountered.
2026-06-10 09:04:06,410 INFO input frame rate=25
/opt/conda/envs/txt2wav/lib/python3.10/site-packages/torch/nn/utils/weight_norm.py:144: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
/opt/conda/envs/txt2wav/lib/python3.10/site-packages/torch/cuda/__init__.py:1061: UserWarning: Can't initialize NVML
  raw_cnt = _raw_device_count_nvml()
text.cc: festival_Text_init
open voice lang map failed
2026-06-10 09:04:10,356 INFO use ttsfrd frontend
  0%|                                                                                                                                        | 0/1 [00:00<?, ?it/s]2026-06-10 09:04:23,791 WARNING synthesis text 高管也通过电话、短信、微信等方式对报道j予好评。 too short than prompt text You are a helpful assistant.<|endofprompt|>希望你以后能够做的比我还好呦。, this may lead to bad performance
/opt/conda/envs/txt2wav/lib/python3.10/site-packages/torchaudio/__init__.py:86: UserWarning: The 'backend' parameter is not used by TorchCodec AudioDecoder.
  return load_with_torchcodec(
2026-06-10 09:04:25,022 INFO synthesis text 高管也通过电话、短信、微信等方式对报道j予好评。
/code/pipeline/lib/CosyVoice/cosyvoice/cli/model.py:103: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  with self.llm_context, torch.cuda.amp.autocast(self.fp16 is True and hasattr(self.llm, 'vllm') is False):
/code/pipeline/lib/CosyVoice/cosyvoice/cli/model.py:426: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  with torch.cuda.amp.autocast(self.fp16):
